glossary.md

```diff
@@ -1,7 +1,21 @@
 # Glossary
 
+## Derivative
+
+A derivative is a contract that derives its value from the performance of an underlying (e.g., an interest rate).
+
+## Future Contract
+
+A futures contract is a bilateral agreement to buy or sell a particular asset at a predetermined price at a specified time in the future. The buyer (seller) of a futures contract is taking on the obligation to buy (sell) and receive (deliver) the underlying asset when the futures contract expires.
+
+## Notional
+
+Notional value is the total underlying amount of a derivatives trade.
+
+## Settlement
+
+Settlement is the delivery of assets to complete trades.
+
 ## Tenor
 
-Tenor is the amount of time it takes for a financial contract to expires. A
-yield swap tenor of 10 days means that it takes 10 days for the given yield swap
-to mature.
+Tenor is the duration of a financial contract.
```

Comment:
FYI I provided https://github.com/yieldswap-protocol/documentation/pull/8 for the glossary changes. Let's figure out this section over in the other PR. Once it gets merged I can update the PR here. From there we can iterate and incorporate all the missing pieces one by one. Hope to make this a smooth sailing for everyone.